### ISSUE TITLE
Paused deployments are incorrectly identified in Prefect 3.0

### ISSUE THREAD

**Issue by ubiquitousbyte** [PrefectHQ/prometheus-prefect-exporter#46]
### Describe the bug a clear and concise description of what the bug is.

The exporter incorrectly identifies paused deployments in Prefect 3.0. The exporter fetches all deployments by issuing the following query against the server:
```bash
curl --location 'http://<prefect-server-url:port>/api/deployments/filter' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--data '{
    "offset": <some-offset>,
    "limit": <some-limit>
}'
```

It then creates the `prefect_info_deployment` metric and adds an `is_schedule_active` label by taking the value directly from the output dictionary in the response - this can be seen [here](https://github.com/PrefectHQ/prometheus-prefect-exporter/blob/ee961e87c10b9ea3914ff41396761f5a385530eb/metrics/metrics.py#L131). The Grafana dashboard uses this to filter for paused deployments via the following Prometheus query:
```prefect_info_deployment{is_schedule_active="False"} == 1```.

In Prefect 3.0, the `is_schedule_active` key is no longer present in the response, so the label value is always `null`. Instead, deployments now have a boolean `paused` key that should be used as a fallback, e.g 
```python
str(deployment.get("is_schedule_active", str(deployment.get("paused", "null"))))
``` 

### Application name

prometheus-prefect-exporter

### Application version

1.5.0

### What you expected to happen?

To preserve backwards compatibility, the `is_schedule_active` label should be set to the value of the `paused` key in the response, instead of the `is_schedule_active` key, which does not exist anymore.

### How to reproduce it?
Run a local Prefect server along with Prometheus, the exporter and Grafana. Create a deployment without a schedule. Import the dashboard. ```prefect_info_deployment{is_schedule_active="False"} == 1``` reports 0 even though there is a paused deployment

**Comment by mitchnielsen** [comment]
Thanks for reaching out @ubiquitousbyte! Appreciate the thorough issue details - are you interested in contributing the fix? If not, no worries - we'll try to get to this within the next 2 weeks.

**Comment by ubiquitousbyte** [comment]
As much as I would love to get involved in this, I won't be able to do so in the next two weeks. Thank you for picking it up.

**Comment by mitchnielsen** [comment]
With https://github.com/PrefectHQ/prometheus-prefect-exporter/pull/47, I'll close this for now. I'll cut a release for this as well. Reach out if you run into any problems with it 👍🏼

**Comment by mitchnielsen** [comment]
Available in 1.6.0:

```
$ helm search repo prefect/prometheus-prefect-exporter -l | head -n 2
NAME                                    CHART VERSION           APP VERSION     DESCRIPTION
prefect/prometheus-prefect-exporter     2024.10.17192636        1.6.0           A Helm chart to deploy Prometheus Prefect Exporter
```
